entries/p/personalcapital.com.json

```diff
@@ -4,11 +4,13 @@
     "url": "https://www.personalcapital.com",
     "tfa": [
       "call",
-      "email"
+      "email",
+      "sms",
+      "totp"
```

Comment:
According to https://support.personalcapital.com/hc/en-us/articles/360056729413 it only works with Personal Capital app. Can you confirm this?

Reply:
It does work with the Personal Capital app using either Push flow or totp code. The generated QR code shown in enrollment however is RFC compliant and can be also registered with any totp app.

Reply:
Okay so it sounds like both `totp` and `custom-software` should be set.
See [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties) for more info.

Reply:
Fair point, pushed update.